doc/classes/Tree.xml

```diff
@@ -107,13 +121,26 @@
 			</return>
 			<description>
 				Returns the currently edited item. Can be used with [signal item_edited] to get the item that was modified.
-				[codeblock]
+				[codeblocks]
+				[gdscript]
 				func _ready():
 				    $Tree.item_edited.connect(on_Tree_item_edited)
 
 				func on_Tree_item_edited():
 				    print($Tree.get_edited()) # This item just got edited (e.g. checked).
-				[/codeblock]
+				[/gdscript]
+				[csharp]
+				public override void _Ready()
+				{
+				    GetNode&lt;Tree&gt;("Tree").ItemEdited += OnTreeItemEdited;
```

Comment:
You can't `+=` with a method.

Reply:
Where is the difference to https://github.com/godotengine/godot/pull/43245/files#diff-cb0c9b0a6dec6df64d66682e6b081e0b1141e270f58a6ecaae0cc9dec1738069R200-R202. I thought that was the beauty of the new callables, that you can use normal events

Reply:
Hmm, I need to look into this further...